src/zmq/zmqnotificationinterface.cpp

```diff
@@ -179,3 +181,37 @@ void CZMQNotificationInterface::NotifyTransactionLock(const CTransaction &tx)
         }
     }
 }
+
+void CZMQNotificationInterface::NotifyGovernanceVote(const CGovernanceVote &vote)
+{
+    for (std::list<CZMQAbstractNotifier*>::iterator i = notifiers.begin(); i != notifiers.end(); )
+    {
+        CZMQAbstractNotifier *notifier = *i;
+        if (notifier->NotifyGovernanceVote(vote))
```

Comment:
This syntax is ... ok I guess -- starting to get into personal preference a bit, but I really prefer to be able to use range-based for which is available as of C++11, and eliminate the if/else condition. Unfortunately that's not always the case, so I'd have to look into how `std::list` accepts an erasure to know if range-based for would work here.